week04/index.html

```diff
@@ -0,0 +1,163 @@
+            if(computer === human){
+                document.getElementById("robotimg").src = "./img/scissor.png";
+                //비겼을 때
+                const text = document.getElementsByClassName("text");
+                for (let i = 0; i < text.length; i++) {
+                    text[i].innerText = "비김";
+                }
+            }
+            else if(human > computer){
+                const humanScore = parseInt(humanscore.innerText);
+                humanscore.innerText = humanScore+1;
+                console.log("승리");
+                //승리 출력
+                const text = document.getElementsByClassName("text");
+                for (let i = 0; i < text.length; i++) {
+                    text[i].innerText = "승";
+                }
+                document.getElementById("robotimg").src = "./img/paper.png";
+            }
+            else if(human < computer){
+                //패배 출력
+                const text = document.getElementsByClassName("text");
+                for (let i = 0; i < text.length; i++) {
+                    text[i].innerText = "패";
+                }
+                const computerScore = parseInt(computerscore.innerText);
+                computerscore.innerText = computerScore+1;
+                document.getElementById("robotimg").src = "./img/rock.png";
+            }
+        };
+
+        function paper_button() {
+            document.getElementById("img").src = "./img/paper.png";
+            human = 2;
+            const computer = Math.floor(Math.random() * 3) + 1;
+            console.log(computer);
+            /*컴퓨터와 비교 후 점수 올리고 내리고*/
+            if(computer === human){
+                document.getElementById("robotimg").src = "./img/paper.png";
+                //비겼을 때
+                const text = document.getElementsByClassName("text");
+                for (let i = 0; i < text.length; i++) {
+                    text[i].innerText = "비김";
+                }
+            }
+            else if(human > computer){
+                const humanScore = parseInt(humanscore.innerText);
+                humanscore.innerText = humanScore+1;
+                console.log("승리");
+                //승리 출력
+                const text = document.getElementsByClassName("text");
+                for (let i = 0; i < text.length; i++) {
+                    text[i].innerText = "승";
+                }
+                document.getElementById("robotimg").src = "./img/rock.png";
+            }
+            else if(human < computer){
+                //패배 출력
+                const text = document.getElementsByClassName("text");
+                for (let i = 0; i < text.length; i++) {
+                    text[i].innerText = "패";
+                }
+                const computerScore = parseInt(computerscore.innerText);
+                computerscore.innerText = computerScore+1;
+                document.getElementById("robotimg").src = "./img/scissor.png";
+            }
+        };
+        
```

Comment:
1. 이벤트에 관한 내용은 JavaScript와 관련이 있으므로, js 부분에서 한 번에 관리하는 것이 좋습니다. EventListener를 추가합니다. 
2. onclick을 직접적으로 사용하여 함수를 추가하는 것은 제한적이라, 일반적으로 EventListener를 사용합니다.
```suggestion
        scissor_btn.addEventListener("click", () => {scoreHandler(1)});
        rock_btn.addEventListener("click", () => {scoreHandler(2)});
        paper_btn.addEventListener("click", () => {scoreHandler(3)});
        reset_btn.addEventListener("click", resetScore);
```